src/activities/melody/Melody.qml

```diff
@@ -58,8 +60,6 @@ ActivityBase {
         }
 
         onStart: {
-            items.backgroundMusicStatus = ApplicationSettings.isBackgroundMusicEnabled
```

Comment:
shouldn't it be removed from the items object too?

Reply:
oops, sorry I didn't notice it. Removed now :+1: